hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/marker/TimelineServerBasedWriteMarkers.java

```diff
@@ -132,6 +148,25 @@ public Set<String> allMarkerFilePaths() {
     }
   }
 
+  @Override
+  public void createMarkerDir() throws HoodieIOException {
```

Comment:
we should make timeline based markers work really well IMO. In cloud storage, direct markers can have same access scaling issues that MT solves for e.g